pyalertme/node.py

```diff
@@ -22,9 +22,18 @@ def __init__(self, callback=None):
         self.addr_short = b''
 
         # Type Info
+        self.nodeId = None
+        self.nodeEui64 = None
+        self.mfgId = None
+        self.deviceType = None
+        self.appRelease = None
+        self.appVersion = None
+        self.hwMinorVersion = None
+        self.hwMajorVersion = None
         self.type = None
+        self.state = None
         self.version = None
-        self.manu = None
+        self.manu_string = None
```

Comment:
Minor comment:
Fine for now but we are starting to mix camelCase variables with under_score style variable names.
To date the majority of PyAlertMe is underscore style (I did this as its code started its origins matching the python-xbee library), but we are introducing camelCase here (which to be honest I do prefer generally).

Leave as is for now. In time we may flip the whole library over to camelCase?

Reply:
I prefer camelCase - shorter